auv_sim/auv_sim_description/auv_common_sim_descriptions/urdf/sensors/sonar.urdf.xacro

```diff
@@ -22,9 +22,9 @@
             </vertical>
           </scan>
           <range>
-            <min>0.5</min>
-            <max>30.0</max>
-            <resolution>0.15</resolution>
+            <min>0.3</min>
```

Comment:
Or course, just as the update rate, these can be parameters as well. But I think these changes are for a future PR in order to keep the context of this PR small and addressable enough.

Reply:
like you said, there are a lot of hard coded values/specifications in the sensors that could be parameters, this may be a future task